VungleAdapter/src/main/java/com/chartboost/mediation/vungleadapter/VungleAdapter.kt

```diff
@@ -130,19 +93,15 @@ class VungleAdapter : PartnerAdapter {
     }
 
     /**
-     * A map of Chartboost Mediation's listeners for the corresponding load identifier.
+     * A lambda to call for successful Vungle ad shows.
      */
-    private val listeners = mutableMapOf<String, PartnerAdListener>()
+    private var onPlaySuccess: () -> Unit = {}
 
     /**
-     * Track the Vungle ad markup for fullscreen ad load --> ad show cycle, keyed by the Vungle placement ID.
+     * A lambda to call for failed Vungle ad shows.
      */
-    private var adms: MutableMap<String, String?> = mutableMapOf()
-
-    /**
-     * Set holding all currently showing Vungle banner placements.
-     */
-    private val showingBanners: MutableSet<String> = mutableSetOf()
+    private var onFailedToPlay: (baseAd: BaseAd, error: VungleError) -> Unit =
+        { _: BaseAd, _: VungleError -> }
```

Comment:
They sould be `onPlaySuccess` and `onPlayFailure` OR `onPlaySucceeded` and `onPlayFailed` for consistency. Or `playback` is fine instead of `play` as well.

Reply:
Yeah, past sense sounds more appropriate. 👍🏼